.changeset/slow-waves-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+"@baseapp-frontend/authentication": patch
+---
+
+BA-1232-ba-fe-baseapp-pre-authenticated-link-shorten-links
```

Comment:
just a suggestion, we usually do a `yarn version-packages` before committing because it's easier to edit the changelog and alter description if needed